v-next/hardhat/templates/mocha-ethers/test/Lock.ts

```diff
@@ -0,0 +1,153 @@
+import hre from "@ignored/hardhat-vnext";
+import { NetworkConnection } from "@ignored/hardhat-vnext/types/network";
+
+// TODO: Chai as promised support needs to be added either directly
+// or through the updated `chai-matchers` package.
+import * as chai from "chai";
+import chaiAsPromised from "chai-as-promised";
+chai.use(chaiAsPromised);
+const { expect } = chai;
+
+describe("Lock", function () {
+  let connection: NetworkConnection<"l1">;
```

Comment:
Now that the connection is not ambient to the global context we need a new pattern for tests.
I have no strong feelings here so opted for a shared `connection` that is explicitly typed for the expected network connection and that is setup for each test.

I suspect we may want to go with something other than `connection` as the variable name. Maybe `networkConnection` is less ambiguous?

Reply:
+1 to `networkConnection`. That's the name I used in the `network-manager` tests.

Reply:
I'm not sure we need to reset it between tests, though.